pulp_ansible/app/tasks/collections.py

```diff
@@ -1052,6 +1050,15 @@ async def run(self):
             # Ensure PR 'total' is correct before stage finishes
             pb.total = pb.done
 
+        tasks = []
+        msg = _("Parsing Namespace Metadata")
+        async with ProgressReport(message=msg, code="sync.parsing.namespace", total=0) as pr:
+            self.parsing_namespace_progress_bar = pr
+            for namespace, namespace_sha in self.namespace_shas.items():
+                tasks.append(loop.create_task(self._add_namespace(namespace, namespace_sha)))
+            await asyncio.gather(*tasks)
+            pr.total = pr.done
```

Comment:
```suggestion
```
Since we know the total before hand we don't have to set it to done. In fact the amount we are going to add vs the amount that did get added can differ, so maybe let it be reflected in the `done` vs `total` amounts.

Reply:
I kept this because I only increment when the download succeeds,
so this would just the total of downloads